src/users/usersActions.js

```diff
@@ -7,8 +7,8 @@ import { PRESENCE_RESPONSE } from '../actionConstants';
 import { getAuth, tryGetAuth } from '../selectors';
 import { isPrivateOrGroupNarrow } from '../utils/narrow';
 
-let lastReportPresence = new Date();
-let lastTypingStart = new Date();
+let lastReportPresence = 0;
+let lastTypingStart = 0;
```

Comment:
Hmm. This makes these variables inconsistently-typed: they're a number at first, and later a Date.

Let's make them consistently one or the other. That way I (and the next reader) don't have to wonder what semantics we're giving to a date subtraction between a `Date` and a number. :-)

Based on MDN's doc:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#Syntax
I think if you say `new Date(0)`, that will give a `Date` object referring to the Unix epoch aka the first moment of 1970 in UTC. That's the thing that I *hope* a plain zero (as in this version) ends up working out to as well.